### PR TITLE
feat: allow use custom action when have panic

### PR DESCRIPTION
**Pull request by levanlongktmt**

### CODE REVIEW

File: env/env.go [hunk 1]
```diff
@@ -7,6 +7,17 @@ import (
...
 
+// Custom action when have panic happen
+var FatalHandler func(interface{})
+
+func doPanic(v interface{}) {
+	if FatalHandler != nil {
+		FatalHandler(v)
+	} else {
+		panic(v)
+	}
```
Comment (jwthanh) [inline env/env.go:18]:
```suggestion
	if FatalHandler != nil {
		FatalHandler(v)
	}
	panic(v)
```

File: env/env.go [hunk 2]
```diff
@@ -7,6 +7,17 @@ import (
 	"strconv"
 )
 
+// Custom action when have panic happen
+var FatalHandler func(interface{})
```
Comment (jwthanh) [inline env/env.go:11]:
```suggestion
// PanicHandler is a custom action when panic happens.
var PanicHandler func(interface{})
```

File: env/env.go [hunk 3]
```diff
@@ -17,11 +28,12 @@ func Get(key string, fallback string) string {
 	return val
 }
 
-// MustGet retrieves the value of the environment variable named key. It panics if the variable is not present.
+// MustGet retrieves the value of the environment variable named key. By default it panics if the variable is not present.
+// use `env.FatalHandler` if you wanna use custom handler, such as send to logstash before panic.
```
Comment (jwthanh) [inline env/env.go:32]:
```suggestion
// use `env.PanicHandler` if you wanna implement a custom handler, e.g. send a log to Logstash before panic.
```

File: env/env.go [hunk 4]
```diff
@@ -7,6 +7,16 @@ import (
 	"strconv"
 )
 
+// PanicHandler is a custom action when panic happens.
+var PanicHandler func(interface{})
+
+func doPanic(v interface{}) {
+	if FatalHandler != nil {
+		FatalHandler(v)
```
Comment (jwthanh) [inline env/env.go:15]:
```suggestion
	if PanicHandler != nil {
		PanicHandler(v)
```

**Comment by levanlongktmt** [comment]
Hmm, I will close this PR because it isn't necessary. 
* If we load env when the app starts, https://github.com/caarlos0/env is a better option
* If load in any other place of code, that's bad behavior when we let the app crash.

**Pull request closed without merging**